pom.xml

```diff
@@ -35,7 +35,7 @@
 		<felles.version>1.20221202080911_bcf8f33</felles.version>
 		<prosessering.version>1.20230103132509_cebcdeb</prosessering.version>
 		<start-class>no.nav.familie.klage.ApplicationKt</start-class>
-		<kontrakter.version>2.0_20221229093739_c660777</kontrakter.version>
+		<kontrakter.version>2.0_20230110150447_4edded8</kontrakter.version>
```

Comment:
EF må også oppdatere fagsystemvedtak før vi merger inn denne. Hvis ikke blir det nullpointer i prod 😬

Reply:
Ahaaa - dét har selvsagt kong @halvorbmundal fikset for oss 👑

Reply:
Allerede fiksa
https://github.com/navikt/familie-ef-sak/pull/2008

Reply:
Med nullable
```suggestion
		<kontrakter.version> 2.0_20230111110429_b460008</kontrakter.version>
```